lib/dmv.rb

```diff
@@ -1,16 +1,20 @@
 class Dmv
-
+  attr_reader :facilities,
+              :service
   def initialize
     @facilities = []
+    @service = service 
   end
 
   def add_facility(facility)
     @facilities << facility
   end
 
   def facilities_offering_service(service)
-    @facilities.find do |facility|
-      facility.services.include?(service)
+    select_service = []
+    @facilities.each do |facility|
```

Comment:
This works, but consider using something that could simplify it even more for you, like .find_all